user_management/ui/exceptions.py

```diff
@@ -0,0 +1,7 @@
+from django.http import Http404
+from django.utils.translation import ugettext_lazy as _
+
+
+class InvalidExpiredToken(Http404):
+    """Exception to confirm an account."""
+    message = _('Invalid or expired token.')
```

Comment:
The API version returns a `400`.  Unfortunately I can't find a way to generate a HTTP400 exception from a normal Django site - any ideas?